### PR TITLE
Add gsutil to doctor

### PR DESCRIPTION
**Pull request by wgrzelak**

### CODE REVIEW

File: scripts/doctor.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ def main():
       gcloud_project=(check_gcloud_default_project, ['gcloud_login']),
       kubectl=check_kubectl,
       kubectl_nodes=(check_kubectl_nodes, ['kubectl']),
-      crd=(check_crd, ['kubectl_nodes']))
+      crd=(check_crd, ['kubectl_nodes']),
+      gsutil=check_gsutil)
```
Comment (huyhg) [inline scripts/doctor.py]: this should be after `['gcloud_login']`

File: scripts/doctor.py [hunk 2]
```diff
@@ -48,10 +48,10 @@ def main():
       gcloud=check_gcloud,
       gcloud_login=(check_gcloud_login, ['gcloud']),
       gcloud_project=(check_gcloud_default_project, ['gcloud_login']),
+      gsutil=check_gsutil,
```
Comment (huyhg) [inline scripts/doctor.py]: when I said after, I meant to include gcloud_loging as a dependency. The order here doesn't matter. This should be written as:

```python
gsutil=(check_gsutil, ['gcloud_login'])
```

Comment (wgrzelak) [reply scripts/doctor.py]: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/375

**Pull request merged**